image/tarexport/tarexport.go

```diff
@@ -30,18 +32,36 @@ type tarexporter struct {
 	loggerImgEvent LogImageEvent
 }
 
+type ocitarexporter struct {
+	te   *tarexporter
+	name string
```

Comment:
@runcom Perhaps, `name` becomes an argument to `Load`? I'm a little concerned that we have naming at this level. This should just import the image and then it should be marked with a name in the reference store.